### DISCUSSION TITLE
Compatible with other pencils?

### DISCUSSION THREAD

**Discussion by AnimaAnimals** [keiwando/toonsquid#116] [Q&A]
I’m considering buying it but i don't use an Apple pencil, is it possible to use the app with other pencils?

**Comment by keiwando** [comment]
Hi!

I have not tested the app with any third-party styluses, so I can currently only guarantee that it works with Apple Pencil and touch inputs.

If the stylus manufacturer claims that the stylus works anywhere that Apple Pencil works, then it should be no problem. If the stylus requires app developers to manually integrate its technology into their app, then the stylus would currently not work with ToonSquid.

I have heard from some users that their non-Apple Pencil stylus worked without issues, but I don't know what exact stylus models those were.

**Comment by kavlor** [comment]
The 3rd party Stylus I have works in Toonsquid but not in Callipeg.
